pkgs/development/python-modules/torchaudio/bin.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchurl
+, isPy37
+, isPy38
+, isPy39
+, patchelf
+, python
+, pytorch-bin
+, pythonOlder
+, pythonAtLeast
+}:
+
+buildPythonPackage rec {
+  version = "0.9.1";
+
+  pname = "torchaudio";
+
+  format = "wheel";
+
+  src =
+    let pyVerNoDot = lib.replaceStrings [ "." ] [ "" ] python.pythonVersion;
+        unsupported = throw "Unsupported system";
+        srcs = (import ./binary-hashes.nix version)."${stdenv.system}-${pyVerNoDot}" or unsupported;
```

Comment:
I think stdenv.system got recently deprecated.

Reply:
Is it top-level system variable? If so, it has changed.

Reply:
CI throws an error with top-level system variable, so I've reverted it.